modules/sdk-core/src/bitgo/utils/tss/eddsa/eddsa.ts

```diff
@@ -64,6 +110,14 @@ export class EddsaUtils extends baseTSSUtils<KeyShare> {
 
     const bitGoToUserPrivateShare = await this.decryptPrivateShare(bitGoToUserShare.privateShare, userGpgKey);
 
+    await this.verifyWalletSignatures(
+      userGpgKey.publicKey,
+      userGpgKey.publicKey,
```

Comment:
pass in the backupGpgKey.publicKey

Reply:
There is no `backupGpgKey.publicKey` in this function available. Do you want me to change the entire wallet creation flow so that we use separate gpg keys for the user and the backup even though it's the same party?

Reply:
In that case, are you ok if I do this in a separate PR? This change is going to bleed into the ecdsa code as well so I'd rather keep this as a separate work item.

Reply:
yeah, i'm fine with that, as long as we track this in a ticket and not forget about it.

Reply:
Ticket created and DMed to you.